From fe62614c47ce7cd9f24e4ed76f8ed4f80e451e77 Mon Sep 17 00:00:00 2001
From: Shunqing Chen <csq@rock-chips.com>
Date: Wed, 7 Nov 2018 16:33:24 +0800
Subject: [PATCH] Revert "arm64: dts: rockchip: rk3399pro-evb: enable bq25703"

This reverts commit 7e9c1889d88509cf6ba5f3b0bc17b8e7543bf80c.

The rk3399pro evb has no battery.

Change-Id: I6d5bc6f17c3335923e34e8d05f2dc1a563b09b00
Signed-off-by: Shunqing Chen <csq@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts
index 83e37e45dc7f..2e42a2183196 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts
@@ -641,7 +641,7 @@
 		ti,otg-current = <500000>;
 		ti,input-current = <500000>;
 		pd-charge-only = <0>;
-		status = "okay";
+		status = "disabled";
 	};
 };
 
-- 
2.35.3

